components/comment/Graffitipost.js

```diff
@@ -0,0 +1,239 @@
+    location: "Berlin",
+    image: "/images/gms-train.png",
+    comments: [
+      {
+        comment: "Fuck DB",
+        graffitiPostId: "u-030Lbc",
+      },
+    ],
+  },
+  {
+    id: "g-12",
+    userID: "u-213Lbc",
+    location: "Hamburg",
+    image: "/images/bates-wall.png",
+    comments: [
+      {
+        comment: "Still standing",
+        graffitiPostId: "u-213Lbc",
+      },
+    ],
+  },
+  {
+    id: "g-040Lba",
+    userID: "u-040Lbc",
+    location: "Hamburg",
+    image: "/images/srf-crew.png",
+    comments: [
+      {
+        comment: "SRF for Life",
+        graffitiPostId: "u-040Lbc",
+      },
+    ],
+  },
+  {
+    id: "g-040Lbc",
+    userID: "u-040Lbc",
+    location: "Hamburg",
+    image: "/images/emra.png",
+    comments: [
+      {
+        comment: "SRFFFF",
+        graffitiPostId: "g-040Lbc",
+      },
+    ],
+  },
+  {
+    id: "g-318Lca",
+    userID: "u-318Lca",
+    location: "Itzehoe",
+    image: "/images/quer.png",
+    comments: [
+      {
+        comment: "Yooo where is that",
+        graffitiPostId: "u-318Lca",
+      },
+    ],
+  },
+  {
+    id: "g-318Lci",
+    userID: "u-318Lci",
+    location: "Itzehoe",
+    image: "/images/wear-alsen.png",
+    comments: [
+      {
+        comment: "Yooo nice Piece",
+        graffitiPostId: "u-318Lci",
+      },
+    ],
+  },
+  {
+    id: "g-318Lcid",
+    userID: "u-318Lci",
+    location: "Itzehoe",
+    image: "/images/wear-image.png",
+    comments: [
+      {
+        comment: "Bro Killing it",
+        graffitiPostId: "u-318Lci",
+      },
+    ],
+  },
+  {
+    id: "g-318Lcidad",
+    userID: "u-318Lciad",
+    location: "Itzehoe",
+    image: "/images/asie-wagon.png",
+    comments: [
+      {
+        comment: "Looks Hella Good",
+        graffitiPostId: "u-318Lci",
+      },
+    ],
+  },
+  {
+    id: "g-318Lcidaded",
+    userID: "u-318Lciad",
+    location: "Itzehoe",
+    image: "/images/asie-wall.png",
+    comments: [
+      {
+        comment: "Looks Fire",
+        graffitiPostId: "u-318Lciad",
+      },
+    ],
+  },
+];
+
+const comments = [];
+
+export default function Graffitis() {
+  const [tags, setTags] = useState([]);
+
+  function handleAddTag(newTag) {
+    setTags([newTag, ...tags]);
+  }
+  return (
+    <>
+      {graffitiPosts.map((graffiti) => {
+        const currentUser = users.find((user) => user.id === graffiti.userID);
+
+        return (
+          <>
+            <section key={graffiti.id}>
+              <h2>Post by {currentUser.name}</h2>
+              <p>Location: {graffiti.location}</p>
+              <div
+                style={{
+                  display: "flex",
+                  justifyContent: "center",
+                }}
+              >
+                <Image
+                  src={graffiti.image}
+                  alt={`Graffiti from ${currentUser.name}`}
+                  width={250}
+                  height={250}
+                />
+              </div>
+              <section
```

Comment:
this section appears to only serve a styling purpose and has no semantic purpose here. You should replace it with a div